tests/pages/pmmInventoryPage.js

```diff
@@ -4,27 +4,30 @@ const assert = require('assert');
 module.exports = {
   url: 'graph/inventory?orgId=1',
   fields: {
+    agentsLink: '//li[contains(text(),"Agents")]',
+    agentsLinkOld: '//a[contains(text(), "Agents")]',
+    deleteButton: '//span[contains(text(), "Delete")]',
+    externalExporter: locate('td').withText('External exporter'),
+    forceModeCheckbox: 'span[data-qa="force-field-label"]',
     iframe: '//div[@class="panel-content"]//iframe',
     inventoryTable: '//table',
-    inventoryTableRows: '//table//tr',
     inventoryTableColumn: '//table//td',
-    agentsLink: '//li[contains(text(),"Agents")]',
+    inventoryTableRows: '//table//tr',
+    mongoServiceName: locate('td').withText('mongodb'),
+    mysqlServiceName: locate('td').withText('ps_group_rep'),
     nodesLink: '//li[contains(text(),"Nodes")]',
-    pmmServicesSelector: '//li[contains(text(),"Services")]',
-    agentsLinkOld: '//a[contains(text(), "Agents")]',
     nodesLinkOld: '//a[contains(text(), "Nodes")]',
+    pdphsqlServiceName: locate('td').withText('PDPGSQL'),
     pmmAgentLocator: '//table//td[contains(text(), "PMM Agent")]',
-    serviceIdLocatorPrefix: '//table//tr/td[4][contains(text(),"',
-    deleteButton: '//span[contains(text(), "Delete")]',
-    proceedButton: '//span[contains(text(), "Proceed")]',
-    forceModeCheckbox: 'span[data-qa="force-field-label"]',
-    tableCheckbox: 'div[data-qa="select-row"]',
-    tableRow: '//tr[@data-qa="table-row"]',
-    runningStatus: '//span[contains(text(), "RUNNING")]',
-    externalExporter: locate('td').withText('External exporter'),
+    pmmServicesSelector: '//li[contains(text(),"Services")]',
     postgresExporter: locate('td').withText('Postgres exporter'),
     postgresPgStatements: locate('td').withText('QAN PostgreSQL PgStatements'),
     postgresPgstatmonitor: locate('td').withText('QAN PostgreSQL Pgstatmonitor'),
+    proceedButton: '//span[contains(text(), "Proceed")]',
+    runningStatus: '//span[contains(text(), "RUNNING")]',
+    serviceIdLocatorPrefix: '//table//tr/td[4][contains(text(),"',
+    tableCheckbox: 'div[data-qa="select-row"]',
+    tableRow: '//tr[@data-qa="table-row"]',
```

Comment:
since you are refactoring, why not use locate too?

Reply:
@IvanaBrecherova please update other locators to use `locate` except `serviceIdLocatorPrefix`